packages/devtools_app/lib/src/inspector/inspector_controller.dart

```diff
@@ -524,6 +558,7 @@ class InspectorController implements InspectorServiceClient {
       return;
     }
     updateSelectionFromService(firstFrame: false);
+    notifySelectionListeners();
```

Comment:
this call should be removed. updateSelectionFromService is async so the result isn't available here.

Reply:
you are right